.github/workflows/build-opcua-monitoring-broker-container.yml

```diff
@@ -4,104 +4,70 @@ on:
-        container_name: ${{ env.AKRI_COMPONENT }}
-        container_prefix: ghcr.io/project-akri/akri
-        container_registry_base_url: ghcr.io
-        container_registry_username: ${{ secrets.crUsername }}
-        container_registry_password: ${{ secrets.crPassword }}
-        makefile_component_name: ${{ env.MAKEFILE_COMPONENT }}
-        platform: ${{ matrix.arch }}
-        build_rust: "0"
-
-  multi-arch:
-    if: (github.event_name == 'release') || (github.event_name == 'push' && github.ref == 'refs/heads/main') || (startsWith(github.event_name, 'pull_request') && github.event.action == 'closed' && github.event.pull_request.merged == true && github.ref != 'refs/heads/main')
-    needs: per-arch
-    runs-on: ubuntu-latest
-    timeout-minutes: 5
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
 
-    steps:
-    - name: Checkout repo
-      uses: actions/checkout@v3
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v3
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
 
-    - name: Prepare To Install
-      uses: actions/setup-node@v3
+    - name: Docker meta
+      uses: docker/metadata-action@v5
+      id: meta
       with:
-        node-version: 12
-    - name: Install Deps
-      run: |
-        yarn install
-        yarn add @actions/core @actions/github @actions/exec fs
+        images: ghcr.io/project-akri/akri/opcua-monitoring-broker
+        labels: |
+          org.opencontainers.image.title=akri-opcua-monitoring-broker
+          org.opencontainers.image.description=Akri sample opcua monitoring broker
+        tags: |
+          type=ref,event=pr
+          type=semver,pattern={{version}}
+          type=semver,pattern={{major}}.{{minor}}
+          type=raw,value=${{steps.version-string.outputs.version}},enable=${{github.event_name != 'release'}}
 
-    - name: Run Multi-Arch component build for ${{ env.AKRI_COMPONENT }}
-      uses: ./.github/actions/build-component-multi-arch
+    - name: Build and push
+      uses: docker/build-push-action@v5
       with:
-        github_event_name: ${{ github.event_name }}
-        container_name: ${{ env.AKRI_COMPONENT }}
-        container_prefix: ghcr.io/project-akri/akri
-        container_registry_base_url: ghcr.io
-        container_registry_username: ${{ secrets.crUsername }}
-        container_registry_password: ${{ secrets.crPassword }}
-        makefile_component_name: ${{ env.MAKEFILE_COMPONENT }}
\ No newline at end of file
+        context: .
+        push: ${{ github.event_name != 'pull_request' }}
+        tags: ${{ steps.meta.outputs.tags }}
+        labels: ${{ steps.meta.outputs.labels }}
+        file: build/containers/Dockerfile.opcua-monitoring-broker
+        platforms: linux/amd64,linux/arm64,linux/arm/v7
```

Comment:
Can you refresh me on why buildx does not support arm v8?

Reply:
Hmm AFAIK, `arm64v8 == aarch64 == arm64` in docker nomenclature.